### PR TITLE
Add post about JS IPFS 0.34 release

### PR DESCRIPTION
**Pull request by alanshaw**

### CODE REVIEW

**Comment by hugomrdias** [comment]
looks awesome ship it 🚀

File: content/post/64-js-ipfs-0.34.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+To help you craft your artisanal CIDs we've introduced a `--cid-base` option to a bunch of CLI commands (and `?cid-base` option to a bunch of HTTP API endpoints). Check it out:
+
+```sh
+jsipfs add file.txt --cid-base=base32
+added bafybeibns4lrebrxaymvyshgmki5biwh6cd53idxfpen6ysomzrywtsm44 file.txt
+```
+
+## 💪 Deps got upgrades!
+
+We got you covered with [WebUI 2.3](https://github.com/ipfs-shipyard/ipfs-webui/releases/tag/v2.2.0) (quic support, responsive navbar, and a language selector), [libp2p 0.24](https://blog.ipfs.io/55-js-libp2p-0-24/) and many other upgrades giving you a faster and smaller JS IPFS.
```
Comment (lidel) [inline content/post/64-js-ipfs-0.34.md]:
```suggestion
We got you covered with [WebUI 2.3](https://github.com/ipfs-shipyard/ipfs-webui/releases/tag/v2.3.3) (quic support, responsive navbar, and a language selector), [libp2p 0.24](https://blog.ipfs.io/55-js-libp2p-0-24/) and many other upgrades giving you a faster and smaller JS IPFS.
```

File: content/post/64-js-ipfs-0.34.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+- Check the issues with the `help wanted` label at the Ready column in our waffle board - https://waffle.io/ipfs/js-ipfs?label=help%20wanted
+- Join an IPFS All Hands, introduce yourself and let us know where you would like to contribute - https://github.com/ipfs/team-mgmt/#weekly-ipfs-all-hands
+- Hack with IPFS and show us what you made! The All Hands call is also the perfect venue for demos, join in and show us what you built
+- Join the discussion at http://discuss.ipfs.io/ and help users finding their answers.
+- Join the [⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽](https://github.com/ipfs/team-mgmt/issues/650) and be part of the Sprint action!
+
+# ⁉️ Do you have questions?
+
+The best place to ask your questions about IPFS, how it works and what you can do with it is at [discuss.ipfs.io](http://discuss.ipfs.io). We are also available at the #ipfs channel on Freenode.
```
Comment (lidel) [inline content/post/64-js-ipfs-0.34.md]:
```suggestion
The best place to ask your questions about IPFS, how it works and what you can do with it is at [discuss.ipfs.io](http://discuss.ipfs.io). We are also available at the `#ipfs` channel on Freenode.
```

File: content/post/64-js-ipfs-0.34.md (hunk 1 above)
Comment (alanshaw) [reply content/post/64-js-ipfs-0.34.md]: I left it as that one because that's the release notes with the interesting stuff in it.

**Pull request merged**